test/fixed_map_perf_test.cpp

```diff
@@ -1,89 +0,0 @@
-#include "fixed_containers/consteval_compare.hpp"
-#include "fixed_containers/fixed_index_based_storage.hpp"
-#include "fixed_containers/fixed_map.hpp"
-#include "fixed_containers/fixed_red_black_tree_nodes.hpp"
-
-#include <benchmark/benchmark.h>
-
-#include <array>
-#include <cstddef>
-#include <functional>
-#include <map>
-#include <type_traits>
-
-namespace fixed_containers
-{
-namespace
-{
-using V = std::array<std::array<int, 3>, 30>;
-constexpr std::size_t CAP = 130;
-
-template <class K, class V, std::size_t MAXIMUM_SIZE>
-using CompactPoolFixedMap =
-    FixedMap<K,
-             V,
-             MAXIMUM_SIZE,
-             std::less<int>,
-             fixed_red_black_tree_detail::RedBlackTreeNodeColorCompactness::EMBEDDED_COLOR,
-             FixedIndexBasedPoolStorage>;
-static_assert(std::is_same_v<FixedMap<int, V, CAP>, CompactPoolFixedMap<int, V, CAP>>);
-
-template <class K, class V, std::size_t MAXIMUM_SIZE>
-using CompactContiguousFixedMap =
-    FixedMap<K,
-             V,
-             MAXIMUM_SIZE,
-             std::less<int>,
-             fixed_red_black_tree_detail::RedBlackTreeNodeColorCompactness::EMBEDDED_COLOR,
-             FixedIndexBasedContiguousStorage>;
-
-template <class K, class V, std::size_t MAXIMUM_SIZE>
-using DedicatedColorBitPoolFixedMap =
-    FixedMap<K,
-             V,
-             MAXIMUM_SIZE,
-             std::less<int>,
-             fixed_red_black_tree_detail::RedBlackTreeNodeColorCompactness::DEDICATED_COLOR,
-             FixedIndexBasedContiguousStorage>;
-
-template <class K, class V, std::size_t MAXIMUM_SIZE>
-using DedicatedColorBitContiguousFixedMap =
-    FixedMap<K,
-             V,
-             MAXIMUM_SIZE,
-             std::less<int>,
-             fixed_red_black_tree_detail::RedBlackTreeNodeColorCompactness::DEDICATED_COLOR,
-             FixedIndexBasedContiguousStorage>;
-
-// The reference boost-based fixed_map (with an array-backed pool-allocator) was at 51000
-// at the time of writing.
-static_assert(consteval_compare::equal<50992, sizeof(FixedMap<int, V, CAP>)>);
-static_assert(consteval_compare::equal<50992, sizeof(CompactPoolFixedMap<int, V, CAP>)>);
-static_assert(consteval_compare::equal<50992, sizeof(CompactContiguousFixedMap<int, V, CAP>)>);
-static_assert(consteval_compare::equal<52032, sizeof(DedicatedColorBitPoolFixedMap<int, V, CAP>)>);
-static_assert(
```

Comment:
These memory tests were dropped along with the perf tests.
Maybe move them to `fixed_map_test.cpp` OR repurposes (rename) this file for memory tests?